src/Helper/Helper_Data.php

```diff
@@ -291,6 +314,7 @@ public function get_localised_script_data( Helper_Abstract_Options $options, Hel
 				'deleteFontError'                      => esc_html__( 'A problem occurred. Reload the page and try again.', 'gravity-forms-pdf-extended' ),
 				'fontFileMissing'                      => esc_html__( '<strong>Oops!&nbsp;</strong>Font file gone missing. Please reupload it again.', 'gravity-forms-pdf-extended' ),
 				'fontListError'                        => esc_html__( 'A problem occurred. Reload the page and', 'gravity-forms-pdf-extended' ),
+				'fontListErrorLinkText'                => esc_html__( 'Try again.', 'gravity-forms-pdf-extended' ),
```

Comment:
Combine these two strings together and add `sprintf` placeholders. Then swap the placeholders out for the link HTML in JS.